### DISCUSSION TITLE
Add Authentication Fail

### DISCUSSION THREAD

**Discussion by sizoune** [nylo-core/nylo#182] [Q&A]
hi im trying to add auth using this command

 `dart run scaffold_ui:main auth`,
 
  im choosing supabase as a backend. but when i input the supabase url and press enter, it didnt do anything?

![image](https://github.com/user-attachments/assets/7367ff7d-0cb1-4264-ac9e-334ff8524805)

my pc OS is Windows 11. and im using Android Studio latest stable version

**Comment by agordn52** [comment]
Hi @sizoune,

Could you try this in Android studio from the terminal?

**Reply by sizoune** [reply]
still same, not doing anything, but its ok, i just follow their documentation,

hi i found bug, in Forms, when i use picker

```
  Field("Kegawatdaruratan",
            header: const Text("Jenis Kegawatdaruratan"),
            style: "default",
            cast: FormCast.picker(
              options: kegawatdaruratanOptions ?? [],
            ),
            validate: FormValidator.contains(kegawatdaruratanOptions,
                message: "Kegawatdaruratan tidak boleh kosong!")),
      ];
```

especially in dark mode, the value color is like in the image

![image](https://github.com/user-attachments/assets/7f91b4f0-4b7f-4f0c-a859-7add0f1723ac)

**Reply by agordn52** [reply]
Hi @sizoune, thanks for reporting this. 

It's now fixed in v6.2.3 https://pub.dev/packages/nylo_framework
